mteb/evaluation/evaluators/RetrievalEvaluator.py

```diff
@@ -417,8 +428,8 @@ def __call__(
                 corpus, queries, self.top_k, self.score_function
             )
 
-    @staticmethod
     def evaluate(
+        self,
```

Comment:
why is self needed here?

Reply:
I need it to call the `evaluate_abstention` method from the `RetrievalEvaluator` class:
```
naucs = self.evaluate_abstention(
    results, {**all_ndcgs, **all_aps, **all_recalls, **all_precisions}
)
```